client/app/bundles/common/DashboardPage.tsx

```diff
@@ -128,6 +129,20 @@ const DashboardPageRedirects = (): JSX.Element => {
 
   if (!courses?.length) return <Navigate to="/courses" />;
 
+  if (getUrlParameter('from') === 'auth') {
+    const visitedCourses = courses.filter((c) => !!c.lastActiveAt);
+
+    if (visitedCourses.length > 0) {
+      const latestSeenCourse = courses.reduce((c1, c2) =>
+        new Date(c1.lastActiveAt ?? 0) > new Date(c2.lastActiveAt ?? 0)
```

Comment:
Now that you use `visitedCourses`, then remove `?? 0`

Reply:
fixed